### DISCUSSION TITLE
Training of Faster R-CNN R50 on Detectron2 Pytorch

### DISCUSSION THREAD

**Discussion by wiky231** [facebookresearch/detectron2#4231] [Q&A]
Hi, I'm trying to train a Faster RCNN R50 model through Detectron2 Pytorch on Google Colab. My training dataset consists of about 5000 images and I'm training for 45000 iterations with a batch size of 16. The model is evaluated on the validation set every 500 iterations and I get the info attached below, can I know which of the values below represents mAP@0.5? And what does the APm stand for? Thanks in advance.

`|   AP   |  AP50  |  AP75  |  APs   |  APm   |  APl   |
|:------:|:------:|:------:|:------:|:------:|:------:|
| 23.208 | 38.640 | 24.023 | 10.269 | 22.183 | 25.567 |
[05/12 03:46:13 d2.evaluation.coco_evaluation]: Per-category bbox AP: 
| category        | AP     | category     | AP     | category           | AP     |
|:----------------|:-------|:-------------|:-------|:-------------------|:-------|
| Marine-debris   | nan    | animal_crab  | 22.216 | animal_eel         | 26.423 |
| animal_etc      | 0.886  | animal_fish  | 37.697 | animal_shells      | 9.109  |
| animal_starfish | 0.180  | plant        | 23.591 | rov                | 46.928 |
| trash_etc       | 17.811 | trash_fabric | 15.715 | trash_fishing_gear | 0.000  |
| trash_metal     | 35.573 | trash_paper  | 29.547 | trash_plastic      | 30.209 |
| trash_rubber    | 50.202 | trash_wood   | 25.238 |                    |        |`

**Comment by AliNaqvi110** [comment]
Have you figure out answer kindly share it
